plugin-ssh/che-plugin-ssh-ext-sshkey/src/main/java/org/eclipse/che/ide/ext/ssh/server/UserProfileSshKeyStore.java

```diff
@@ -280,21 +284,29 @@ private String sshKeyAttributeName(String host, int isPrivate) {
 
     @SuppressWarnings("unchecked")
     ProfileDescriptor getUserProfile() throws ApiException, IOException, JsonParseException {
-        final String profileDescriptor = HttpJsonHelper.requestString(profileApiUrl, "GET", null);
-        return JsonHelper.fromJson(profileDescriptor, ProfileDescriptor.class, new TypeToken<ProfileDescriptor>() {}.getType());
+        final String profileDescriptor = HttpJsonHelper.requestString(profileApiUrl, HttpMethod.GET, null);
```

Comment:
HttpJsonHelper can parse response to DTO object. We don't need manual parsing here.